code/modules/mapfluff/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -500,19 +500,22 @@ GLOBAL_VAR_INIT(hhMysteryRoomNumber, rand(1, 999999))
 /obj/effect/landmark/lift_id/hilbert
 	specific_lift_id = HILBERT_TRAM
 
-/obj/effect/landmark/tram/left_part/hilbert
+/obj/effect/landmark/tram/hilbert/left
+	name = "Port"
 	specific_lift_id = HILBERT_TRAM
-	destination_id = "left_part_hilbert"
+	platform_code = 11
```

Comment:
I don't like funny numbers like these, could you make these codes defines like

```dm
#define HILBERT_LEFT 11
```

Also, why does it have to be 11

Reply:
Yeah, made em defines. It doesn't have to be 11, it was a remnant of before I knew if we could reuse codes (we can)

Switched them to 1/2/3.